src/screens/OrgPost/Posts.tsx

```
@@ -0,0 +1,168 @@
+
+  if (isFiltering) {
+    if (!data?.postsByOrganization || data.postsByOrganization.length === 0) {
+      return <NotFound title="post" keyPrefix="postNotFound" />;
+    }
+
+    const filtered = data.postsByOrganization
+      .filter(
+        (post: InterfacePost) =>
+          post &&
+          post.caption &&
+          post.caption.toLowerCase().includes(searchTerm.toLowerCase()),
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)